### DISCUSSION TITLE
recipe_validation_CMIP6.yml giving RuntimeError: (NetCDF:: Not a valid ID)

### DISCUSSION THREAD

**Discussion by AnushaNTNU** [ESMValGroup/ESMValCore#1751] [Q&A] [answered]
Hi 
I am having trouble running an existing recipe ,  ESMValTool/esmvaltool/recipes/recipe_validation_CMIP6.yml. The recipe is exactly the same here, https://github.com/ESMValGroup/ESMValTool/blob/main/esmvaltool/recipes/recipe_validation_CMIP6.yml

I am attaching the log file here. Can anyone help me to know what is the error

[log.txt](https://github.com/ESMValGroup/ESMValCore/files/9733458/log.txt)

**Comment by valeriupredoi** [comment, marked as answer]
hi @AnushaNTNU I just noticed this was left unanswered, but I think you got the answer through https://github.com/ESMValGroup/ESMValTool/discussions/2844 - if that's the case, could you please mark this answer as the Answer? Many thanks :beer: BTW I have also created a :stop_sign: issue for this type of error here https://github.com/ESMValGroup/ESMValTool/issues/2907
